app/assets/scripts/views/emergency.js

```diff
@@ -0,0 +1,221 @@
+'use strict';
+import React from 'react';
+
+import App from './app';
+
+export default class Emergency extends React.Component {
+  render () {
+    return (
+      <App className='page--emergency'>
+        <section className='inpage'>
+          <header className='inpage__header'>
+            <div className='inner'>
+              <div className='inpage__headline'>
+                <div className='inpage__headline-content'>
```

Comment:
This element has no spatial presence and can lead to problems.
![image](https://user-images.githubusercontent.com/1090606/33843326-105f6160-de6b-11e7-89cc-fd91853c1ad6.png)